lib/libc/pthread/pthread_mutexattr_setprotocol.c

```diff
@@ -80,25 +80,26 @@
  *
  ****************************************************************************/
 
-int pthread_mutexattr_setprotocol(FAR pthread_mutexattr_t *attr,
-				  int protocol)
+int pthread_mutexattr_setprotocol(FAR pthread_mutexattr_t *attr, int protocol)
 {
 	svdbg("attr=0x%p protocol=%d\n", attr, protocol);
-	DEBUGASSERT(attr != NULL);
```

Comment:
When we run the test cases for pthread in TC mode, we cannot go forward with DEBUGASSERT as we get this error:
up_assert: Assertion failed at file:pthread/pthread_mutexattr_setprotocol.c line: 86 task: kerneltc
and the rest of the tc's are halted.
If it is not required under code coverage, do we leave out this test case:
ret_chk = pthread_mutexattr_setprotocol(NULL, protocol);
What do you suggest ?